portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,7 +53,38 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
       datastore.put(comment);
+      int max = getRequestNum(request);  
```

Comment:
The HTML form below doesn't set this, so this will always return 10. Is that intentional?

Reply:
yes, so if there's no input for max number, it will be automatically set to 10.

Reply:
I'd suggest declaring a constant at the top of your class of `private static final int defaultMaxComments = 10;` that you use in `getRequestNum` so you can avoid using a magic number and you/future developers can easily change the default number.

Reply:
fixed